lib/Backend/IR.h

```diff
@@ -739,6 +739,7 @@ class BranchInstr : public Instr
     bool                 m_isAirlock : 1;
     bool                 m_isSwitchBr : 1;
     bool                 m_isOrphanedLeave : 1; // A Leave in a loop body in a try, most likely generated because of a return statement.
+    bool                 m_areCmpRegisterFlagsUsedLater : 1; // Indicate that this branch is not the only instr using the register flags set by cmp
```

Comment:
>m_areCmpRegisterFlagsUsedLater [](start = 25, length = 30)

IR::Instrs are not zero allocated.  You need to initialize the flag in the ctor.